packages/agent/src/upgrader-utils.ts

```diff
@@ -0,0 +1,161 @@
+import { normalizeErrorString } from '@medplum/core';
+import { createWriteStream } from 'node:fs';
+import { platform } from 'node:os';
+import { resolve } from 'node:path';
+import { Readable } from 'node:stream';
+
+export type ReleaseManifest = { tag_name: string; assets: { name: string; browser_download_url: string }[] };
+export type SupportedOs = 'windows' | 'linux';
+
+export const UPGRADE_MANIFEST_PATH = resolve(__dirname, 'upgrade.json');
+export const UPGRADER_LOG_PATH = resolve(
+  __dirname,
+  `upgrader-logs-${new Date().toISOString().replace(/:\s*/g, '-')}.txt`
+);
+export const GITHUB_RELEASES_URL = 'https://api.github.com/repos/medplum/medplum/releases';
+export const RELEASES_PATH = resolve(__dirname);
+
+const releaseManifests = new Map<string, ReleaseManifest>();
+
+export function clearReleaseCache(): void {
+  releaseManifests.clear();
+}
+
+export function getOsString(): SupportedOs {
```

Comment:
Could we just use `platform()` directly here, and cut `getOsString()` and `SupportedOs`?